src/main/java/com/bankofspring/model/AccountTransactions.java

```diff
@@ -0,0 +1,41 @@
+package com.bankofspring.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Singleton class to record transactions
+ * @author Malcolm.Murray
+ *
+ */
+public class AccountTransactions {
+
+	private List<AccountTransaction> transactions;
+	
+	private static AccountTransactions instance;
```

Comment:
No, this is just a singleton container for all the transactions - ultimately these should be getting written to the DB instead.